Automattic/wp-calypso
# Jetpack Offer Reset: Show product options in product description (daily or real-time).

#### Changes proposed in this Pull Request

This PR adds the text, "Available options: Real-Time and Daily" to the product description, IF:
- The product offers a real-time and daily option.
- and if the product is not already owned by the user.

#### Testing instructions

- Ensure your sandbox site does't have any Jetpack plans/products purchased.
- Enable the _Offer Reset Flow_ in _A/B tests_.
- Visit the Plans section.

Look at the description section in each of the Plan/Individual Product cards.  Every product/plan that offers a "real-time" and "daily" option will have, "Available options: Real-Time or Daily." appended to the description.  In this case, "Jetpack Backup" and "Jetpack Security" with have the appended text because those are the only plans that offer real-time and daily options at this point.

Screenshots:
![image](https://user-images.githubusercontent.com/11078128/90275283-df95ba80-de16-11ea-905f-116531abccb8.png)

![image](https://user-images.githubusercontent.com/11078128/90275694-8da16480-de17-11ea-917d-c44ebc216139.png)
> Please note:  The "Jetpack Backup" product only has some Lorem Ipsum for the product description. I'm assuming this will be handled in another PR.

Next, purchase Jetpack Backup, reload the Plans page and You should no longer see the appended text ("Available options: Real-Time or Daily.") in the product description of the Jetpack Backup product card. 

Screenshot:
![image](https://user-images.githubusercontent.com/11078128/90276476-d4438e80-de18-11ea-8389-4ce45f87e35b.png)
